scripts/apply_patches.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# these probably won't change
+USER="psy"
+# these will probably change over time for versioning
+#TARGETBRANCH="evt-patched-v1.1"
+TARGETBRANCH="my-local-branch"
+
+# this all assumes patches/${REPO}.patch exists
+# ...for every submodule repo
+
+# assumes we're running from top-level of EVT repo
+# go into submodules directory
+cd submodules
+for REPO in $(ls); do
```

Comment:
We should iterate over an explicit directory list rather than loop over each submodule and assume each submodule has a patch